apps/web-giddh/src/assets/css/global/input-dropdown-field.scss

```diff
@@ -396,13 +382,13 @@ add-company {
     }
 }
 
-.mat-mdc-option {
-    &:focus {
-        &.mdc-list-item {
-            background-color: transparent !important;
-        }
-    }
-}
+// .mat-mdc-option {
```

Comment:
Your not self reviewing pr,

Remove commented code

Reply:
done